openmetadata-ui/src/main/resources/ui/src/components/MyData/Widgets/FeedsWidget/FeedsWidget.component.tsx

```diff
@@ -63,7 +63,15 @@ const FeedsWidget = ({
 
   useEffect(() => {
     if (activeTab === ActivityFeedTabs.ALL) {
-      getFeedData(defaultFilter, undefined, ThreadType.Conversation);
+      getFeedData(
+        defaultFilter,
+        undefined,
+        ThreadType.Conversation,
+        undefined,
+        undefined,
+        undefined,
+        PAGE_SIZE_LARGE
+      );
```

Comment:
Not urgent but I think we should refactor this function to have an single object as an argument to avoid these unnecessary undefined

Reply:
yes, have taken not refactor this!